### ISSUE TITLE
Update to TinyMCE caused multiple interface issues to the Integrated version

### ISSUE THREAD

**Issue by ilovan** [cwrc/CWRC-Writer#359]
the mirror Islandora installation @ cwrc-dev-05.srv.ualberta.ca is set up to automatically update the integrated CWRC-Writer to the latest version on http://apps.testing.cwrc.ca/editor/dev/index.htm
Test object at http://cwrc-dev-05.srv.ualberta.ca/islandora/object/cwrc%3A18c33e64-518f-4a00-8795-7295f7b0bc2c/cwrc-writer#cwrc:18c33e64-518f-4a00-8795-7295f7b0bc2c
Andrew received an account a few days ago, so he should be able to check it out. 

Some of the issues identified at a first glance: 
* absence of tabs
* size of the text area
* editor mode pop-up layout

**Comment by ajmacdonald** [comment]
I don't have permissions to view the test object.

**Comment by ilovan** [comment]
Try now.

**Comment by ajmacdonald** [comment]
Thanks I can now access it.

It looks like this CSS file isn't being included: https://github.com/cwrc/CWRC-Writer/blob/development/src/css/cwrcstyle/jquery-ui.css
Including it should hopefully fix most/all of the styling problems.

**Comment by ilovan** [comment]
Great, thanks, Andrew. 
@jefferya , would it be possible to include the style sheet Andrew mentioned into our dev05 version of CWRC-Writer? This would help us see what styling problems are not covered by this css.

**Comment by jefferya** [comment]
Possible, likely. The addition may get clobbered by DGi's work.

**Comment by ilovan** [comment]
Update: I just brought this issue to the attention of the new dgi project manager and they are aware of the need to integrate the style-sheet specified by Andrew. 
In the meantime, @jefferya , could you please integrate it on dev05 so that we can see if there are any other styling issues that we need to fix?
Thanks.

**Comment by jefferya** [comment]
@ilovan jquery-ui.css has been added to the Islandora integration module. Looks better but I've only taken a quick glance. When there is time, can the change be tested further?

@ilovan Should a pull request be sent to DGi?  Or should the change be tested further?

FYI. Here is the code revision:
https://github.com/cwrc/islandora_cwrc_writer/commit/75f32410e6c8ef9d275864f8366e5db15dbd6695

**Comment by ilovan** [comment]
Thanks, @jefferya .
Andrew, I just tested again at http://cwrc-dev-05.srv.ualberta.ca/islandora/object/cwrc%3A18c33e64-518f-4a00-8795-7295f7b0bc2c/cwrc-writer#cwrc:18c33e64-518f-4a00-8795-7295f7b0bc2c.

The following issues have been fixed: 
* absence of tabs
* editor mode pop-up layout

The size of the text area is still a problem, though. 

Let's try to fix that too and we'll send pull requests for all these fixies in the end, before the next stage of the dgi work begins.
